consumer/dragonboard820c/build/linux-kernel.md

```diff
@@ -12,17 +12,20 @@ for Dragonboard-820c.
 
 The Linux kernel used for DragonBoard 820c can be found in the
 [Linaro Qualcomm Landing Team git repository](https://git.linaro.org/landing-teams/working/qualcomm/kernel.git).
-For now the support for this board is preliminary and can only be found in
-either `release` branches named as `release/db820c/qcomlt-x.y`
-(the latest one being the most up-to-date/recent) or the
-`integration-linux-qcomlt` branch, which is regularly rebased on recent mainline,
+
+There are two main relevant kernel branches that provide the support for this board.
+1. `release` branches named as `release/qcomlt-x.y`
+(the latest one being the most up-to-date/recent)  
+1. the `integration-linux-qcomlt` branch, which is regularly rebased on recent mainline,
 and is used by developers.
 
+Using the `release` branch is recommended for most users. The development branch is meant to be used by kernel developers willing to contribute patch upstream.
+
 Following commands can be used to fetch the latest kernel source:
 
 ```shell
 $ git clone http://git.linaro.org/landing-teams/working/qualcomm/kernel.git
-$ git checkout release/db820c/qcomlt-x.y -b db820c
+$ git checkout -t release/qcomlt-x.y 
```

Comment:
Don't we need `-b` here?

`git checkout -t -b release/qcomlt-x.y`

Reply:
it's not needed, when -b is not used, it will have a good default name

from git help checkout:
           If no -b option is given, the name of the new branch will be derived from the remote-tracking branch, by looking at the local part of the refspec configured for the corresponding remote, and then stripping the initial part up to the "*". This would tell us to use
           "hack" as the local branch when branching off of "origin/hack" (or "remotes/origin/hack", or even "refs/remotes/origin/hack").

Reply:
argh. we need -t qcomlt/release/qcomlt-x.y.. i will update

Reply:
got it! thanks